### PR TITLE
Create Base_Norwegian.lang

### PR DESCRIPTION
**Pull request by Larskrs**
Norwegian language base.

### CODE REVIEW

File: translations/Base_Norwegian.lang [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+second: "sekund"
+seconds: "sekunder"
+settings: "instillinger"
+settingsView.select.action: "Klikk for å se instillinger"
+settingsView.select.indicator: "Denne siden er **valgt**"
+settingsView.title: "Instillinger > %"
+timePickerView.addDay: "Legg til 1 Dag"
+timePickerView.addHour: "Legg til 1 Time"
+timePickerView.addMinute: "Legg til 1 minutt"
+timePickerView.addSecond: "Add 1 sekund"
```
Comment (rlnt) [inline translations/Base_Norwegian.lang:110]: Everything after this line (the line included) still needs translation. Did you forget it?

**Review by rlnt — CHANGES_REQUESTED** [review]
Please finish the translation

File: translations/Base_Norwegian.lang (hunk 1 above)
Comment (Larskrs) [reply translations/Base_Norwegian.lang:110]: Oh, sorry, didn’t notice it. I will finish it as fast as i can!

Comment (Larskrs) [reply translations/Base_Norwegian.lang:110]: @DAmNRelentless finished it!

File: translations/Base_Norwegian.lang [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+timePickerView.decrementAction: "Klikk for å fjerne"
+timePickerView.incrementAction: "Klikk for å legge til"
+timePickerView.removeDay: "Fjern 1 Dag"
+timePickerView.removeHour: "Fjern 1 Time"
+timePickerView.removeMinute: "Fjern 1 Minutt"
+timePickerView.removeSecond: "Fjern 1 Sekund"
+update.clickToAuthenticate: "Klikk på linken nede, for å autentisere gjennom **Discord**:"
+update.remote.notify: "Oppdaterte fra **%from%** til **%to%**"
+update.serverOffline: "Det var et problem gjennom tilkobling to Oppdaterings Serverene våres"
+update.updatedPlugin: "Updated the plugin to **%**, reloading.."
```
Comment (rlnt) [inline translations/Base_Norwegian.lang]: These 3 lines are still missing translation.

**Review by rlnt — CHANGES_REQUESTED** [review]
Please finish the translation.

**Comment by rlnt** [comment]
Lines 122-124 are still missing, take a look at the review @Larskrs

**Review by rlnt — APPROVED** [review]
Thanks for your contribution :)

**Pull request merged**